src/assets/translations/en/main.json

```diff
@@ -3004,6 +3003,11 @@
       "pending": "Your withdrawal of %{amount} %{symbol} is being processed.",
       "complete": "Your withdrawal of %{amount} %{symbol} is complete."
     },
+    "claim": {
+      "cta": "Claim",
```

Comment:
Can we pls remove this and use `common.claim` across the board? 🙏🏽

Reply:
@premiumjibles for a follow-up if we feel like it, going to merge as dis non-blocking